src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -88,6 +88,10 @@ static ExperimentalFeature()
                                         source: EngineSource,
                                         isEnabled: false),
                 */
+                new ExperimentalFeature(name: "PSImplicitRemotingBatching",
+                                        description: "Batch implicit remoting proxy commands to improve performance",
+                                        source: "EngineSource",
+                                        isEnabled: true)
```

Comment:
Please use `isEnabled: false` here. When the feature is enabled in `powershell.config.json`, the feature will be shown as `Enabled`.

Reply:
Will do.